newIDE/app/src/ProjectCreation/NewProjectSetupDialog.js

```diff
@@ -424,236 +449,243 @@ const NewProjectSetupDialog = ({
+                      <Trans>Remix an existing game</Trans>
+                    </Text>
+                    <ExampleStore
+                      onSelectExampleShortHeader={exampleShortHeader => {
+                        onSelectExampleShortHeader(exampleShortHeader);
+                      }}
+                      onSelectPrivateGameTemplateListingData={privateGameTemplateListingData => {
+                        onSelectPrivateGameTemplateListingData(
+                          privateGameTemplateListingData
+                        );
+                      }}
+                      i18n={i18n}
+                      columnsCount={getItemsColumns(windowSize, isLandscape)}
+                      onlyShowGames
+                      rowToInsert={{
+                        row: 2,
+                        element: (
+                          <AIPromptField
+                            onCreateFromAIGeneration={onCreateFromAIGeneration}
+                            isProjectOpening={!!isProjectOpening}
+                            isGeneratingProject={isGeneratingProject}
+                            storageProvider={storageProvider}
+                            saveAsLocation={saveAsLocation}
+                            onGenerationStarted={() =>
+                              setIsGeneratingProject(true)
+                            }
+                            onGenerationEnded={() =>
+                              setIsGeneratingProject(false)
+                            }
+                          />
+                        ),
+                      }}
+                    />
+                  </>
+                )}
+              </ColumnStackLayout>
+            )}
+            {!isOnHomePage &&
+              selectedPrivateGameTemplateListingData &&
+              !isSelectedGameTemplateOwned && (
+                <PrivateGameTemplateInformationPage
+                  privateGameTemplateListingData={
+                    selectedPrivateGameTemplateListingData
+                  }
+                  privateGameTemplateListingDatasFromSameCreator={
+                    privateGameTemplateListingDatasFromSameCreator
+                  }
+                  onGameTemplateOpen={onSelectPrivateGameTemplateListingData}
                 />
               )}
-            </SelectField>
-            {needUserAuthenticationForStorage && (
-              <Paper background="dark" variant="outlined">
-                <Line justifyContent="center">
-                  <CreateProfile
-                    onOpenLoginDialog={authenticatedUser.onOpenLoginDialog}
-                    onOpenCreateAccountDialog={
-                      authenticatedUser.onOpenCreateAccountDialog
-                    }
-                    message={
-                      <Trans>
-                        Create an account to store your project online.
-                      </Trans>
+            {!isOnHomePage && isSelectedGameTemplateOwned && (
```

Comment:
I feel like `isSelectedGameTemplateOwned` means something else, like `isExampleOrOwnedGameTemplateOpened`.
I was troubled by the check on `selectedExampleShortHeader` below and I was already surprised to see that `isSelectedGameTemplateOwned` is true if there is no game template selected

Reply:
yeah, I struggled finding a name for it, I'll think of something else